FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Multitask/DemoTasks/MultitaskMQTTExample.c

```diff
@@ -250,6 +250,17 @@
  */
 #define mqttexampleUNSUBSCRIBE_COMPLETE_BIT          ( 1U << 1 )
 
+/**
+ * @brief The maximum number of loop iterations to wait before declaring failure.
+ *
+ * Each `while` loop that waits for a task notification will wait for
+ * `mqttexampleDEMO_TICKS_TO_WAIT * this amount of ticks before exiting.
```

Comment:
```suggestion
 * `mqttexampleDEMO_TICKS_TO_WAIT * this amount of iterations before exiting.
```

Reply:
I was trying to convey that the total number of ticks to wait is `mqttexampleDEMO_TICKS_TO_WAIT` * this macro. I'll reword that